composer.json

```
@@ -25,12 +25,10 @@
         "php-amqplib/php-amqplib": "~2.4 || ^3",
         "php-console/php-console": "^3.1.3",
         "phpspec/prophecy": "^1.6.1",
-        "phpunit/phpunit": "^8.5",
+        "phpunit/phpunit": "^9",
```

```suggestion
        "phpunit/phpunit": "^9.5",
```
Running PHPUnit 9.4 on PHP 8.1 is a bad idea. 🙃